### PR TITLE
Configuration values for interop-net bootstrap and drand

### LINKED ISSUE #3863: Avoid importing fixtures into main binary
The `fixtures` package is imported to the go-filecoin binary and all the `init()` methods in `constants.go`  execute when running go-filecoin. This package should not be imported into the binary!

### PR DESCRIPTION
**Pull request by anorth**
### Motivation
Adding bootstrap and DRAND config for long-running networks. The values all come from @travisperson's work on interop and testnet deployments.

Changed the default sector size configuration to enable 2KiB sectors so that local nodes can be set-up without hand-editing the conf file. The long-running networks will likely have an option similar to `--devnet-interop` that sets the right value for each network.

### Proposed changes
Closes #3863 (but I'll follow up to make it more robust)

### CODE REVIEW

File: functional-tests/network-deployment/relay_check_test.go [hunk 1]
```diff
@@ -303,12 +303,8 @@ func networkBootstrapPeers(network string) []string {
...
 
 	switch network {
-	case "nightly":
-		return fixtures.DevnetNightlyBootstrapAddrs
-	case "staging":
-		return fixtures.DevnetStagingBootstrapAddrs
-	case "user":
-		return fixtures.DevnetUserBootstrapAddrs
+	case "interop":
+		return networks.InteropNet.Bootstrap.Addresses
```
Comment (anorth) [inline functional-tests/network-deployment/relay_check_test.go:307]: @travisperson we should consider whether these network deployment tests are worth keeping in this form - I'd appreciate your advice (to follow-up).

File: internal/app/go-filecoin/connectors/retrieval_market/provider_test.go [hunk 2]
```diff
@@ -49,7 +50,12 @@ func TestRetrievalProviderConnector_UnsealSector(t *testing.T) {
 	tf.UnitTest(t)
 	ctx := context.Background()
 	sectorID := rand.Uint64()
-	fixtureFile := "../../../../../fixtures/constants.go"
```
Comment (anorth) [inline internal/app/go-filecoin/connectors/retrieval_market/provider_test.go:52]: This blew my mind. I think I've captured the intent of the test without referring to this file now.

**Comment by anorth** [comment]
@travisperson do you have the values for testnet-2 that we can add to a similar network configuration in a follow-up PR?

File: functional-tests/network-deployment/relay_check_test.go (hunk 1 above)
Comment (travisperson) [reply functional-tests/network-deployment/relay_check_test.go:307]: This test can definitely go. We are not using relays.

**Comment by travisperson** [comment]
@anorth Not at the moment.

Min miner is 1TiB, supported sectors will be 32, 64, and there will be different peers, drand will be the same.

I'll have the peer ids later.

**Pull request merged**